tony/README.md

```diff
@@ -0,0 +1,53 @@
+# TonY - TensorFlow on YARN
```

Comment:
Can you add a test_tony.sh? example: https://github.com/GoogleCloudPlatform/dataproc-initialization-actions/blob/master/kafka/test_kafka.py